src/main/java/org/broadinstitute/consent/http/models/elastic_search/DatasetTerm.java

```diff
@@ -0,0 +1,170 @@
+package org.broadinstitute.consent.http.models.elastic_search;
+
+import java.util.List;
+import org.broadinstitute.consent.http.models.ontology.DataUseSummary;
+
+public class DatasetTerm {
+
+  private Integer datasetId;
+  private String datasetIdentifier;
+  private String description;
+  private String studyName;
+  private Integer studyId;
+  private Integer participantCount;
+  private String phenotype;
+  private String species;
+  private String piName;
+  private UserTerm dataSubmitter;
+  private String dataCustodian;
+  private DataUseSummary dataUse;
+  private List<String> dataTypes;
+  private String dataLocation;
+  private String dacName;
+  private Boolean openAccess;
+  private Boolean publicVisibility;
+  private List<UserTerm> approvedUsers;
```

Comment:
In the ticket, it specified just using emails, or names. I figured it would be more robust if we just created a user term, so we could go email, name or id.

Reply:
To meet MVP, we can use just user ids.The UI will (eventually) have a need to filter on datasets that the user already has access to. Since the user id is known at that time, that's the minimum info we need to filter on. More info does provide additional filterable abilities, but we need to explore those when that need arises.

Reply:
So, would you prefer to remove the UserTerm class and just move to IDs?